### ISSUE TITLE
Expecting an event not to be generated

### ISSUE THREAD

**Issue by rhyslbw** [meteor-space/testing-event-sourcing#10]
Since we switched to `containArrayOfStructs` in the `expect` method, it's now not possible to check that an event is not generated. I'm thinking we should just provide an option when declaring the test as either 'contains' or 'matches', as really since i've started thinking about these integration tests as testing the whole bounded context, I _do_ want to assert the exact array of events generated are expected. I'd also vote for that to return as the default behaviour, with the `contains` mode as the option. thoughts?

**Comment by DominikGuzei** [comment]
Hmm … yeah i think the optimal solution would be to test the complete series of events but to be able to either pass an instance or a class. This way it's still quite manageable and you can pass in any level of detail.

**Comment by rhyslbw** [comment]
Yes you're right, that's a good flexible approach. `contains` really encourages bad practice, and while the underlying method still has value, I think in this case it should not be used.
